presto-main/src/main/java/io/prestosql/sql/analyzer/Analyzer.java

```diff
@@ -79,7 +79,7 @@ public Analysis analyze(Statement statement, boolean isDescribe)
     {
```

Comment:
Can you please provide some test for that?

It could be with file based access control and file based group provider, to see if view owner is capable to read from table.

Reply:
Do you have an example of how to set this up? I assume we’d need a product test that uses the Hive connector and the group provider, but I don’t know how to do that.

Reply:
`FileGroupProvider` ?

Reply:
The problem is that `FileGroupProvider` is defined in `presto-password-authenticators` while `FileBasedSystemAccessControl` is defined in `presto-plugin-toolkit`, that makes this test a bit awkard. 

Another approach, preferable by me, would be to change `DistributedQueryRunner` to accept `GroupProviders` so we could simply mock the group provider in tests like we can do with system level access control. Then you could just run simple query.

Reply:
We can extend the DistributedQueryRunner. However, if we cannot test using FileBasedSystemAccessControl and FileGroupProvider other than in product tests, then something is wrong with our non-product tests. Isn't `presto-tests` supposed to give us such capabilities?

Reply:
We shouldn't need `FileBasedSystemAccessControl` for a test if we are doing a `DistributedQueryRunner` since that can also mock the access control. Note that while the code is in `presto-plugin-toolkit`, it's actually part of the core server in `presto-main`, unlike `FileGroupProvider` which is actually a plugin.

Reply:
That is correct